client/src/__tests__/Footer.test.js

```diff
@@ -0,0 +1,27 @@
+import {render, screen, cleanup} from '@testing-library/react';
+import Footer from '../components/Footer';
+import ReactDOM from "react-dom";
+
+
+
+
+afterEach(()=>{
+    cleanup();
+    });
+    
+    test('should render Footer component', () => {
+        render(<Footer/>);
+        const EmailElement = screen.getByRole("link", 'mailto:email@domain.com');
+        expect(EmailElement).toBeInTheDocument();  
+        expect(EmailElement).toBeEnabled();  
+
+        const imageElement = screen.getByRole('img',{class: /contact/i})
+        expect(imageElement).toBeInTheDocument();  
```

Comment:
Good Work at unit Testing

Reply:
Good recommendation. I may look into it if I have time.